src/common/gui/SurgeGUIEditor.cpp

```diff
@@ -51,6 +51,10 @@
 #include <stack>
 #include <unordered_map>
 
+#if WINDOWS
+#include <windows.h>
+#endif
```

Comment:
Curious why?

Reply:
This is for the definitions of `SYSTEMTIME`/`GetSystemTime`. Before, `windows.h` was included implicitly via `SurgeStorage.h`→`CriticalSection.h`.

Reply:
Thanks!
We should move that to std::chrono one day

Reply:
we actually didn't need the time at all so just cleaned it up and pushed